src/Microsoft.AspNetCore.Mvc.ApiExplorer/ApiBehaviorApiDescriptionProvider.cs

```diff
@@ -38,20 +38,20 @@ public void OnProvidersExecuting(ApiDescriptionProviderContext context)
                     continue;
                 }
 
-                foreach (var responseType in CreateProblemResponseTypes(description))
```

Comment:
I'm not sure we should have this class at all in preview1. This is going to create responses without any type information. Does this override what a user would have configured?

Reply:
Discusses offline and we removed this type for preview1.